containers/addresses/EditAddressModal.js

```diff
@@ -15,6 +15,10 @@ import {
     useEventManager
 } from 'react-components';
 
+const RichTextEditorEmptyValue = '<div><br></div>';
```

Comment:
What do you think about putting it in an Array constant?
```suggestion
const EMPTY_VALUES = ['<div><br></div>'];
```